docs/dev/new_check_howto.md

```diff
@@ -426,33 +458,59 @@ sudo -u dd-agent datadog-agent integration install -w /path/to/wheel.whl
 sudo datadog-agent integration install -w /path/to/wheel.whl
 ```
 
-**Windows** (Ensure that your shell session has _administrator_ privileges):
+**Windows Powershell** (Ensure that your shell session has _administrator_ privileges):
```

Comment:
```suggestion
**Windows PowerShell** (Ensure that your shell session has _administrator_ privileges):
```

Reply:
hehe, yeah I noticed this one and pushed up a fix 😅